static/app/components/modals/newDemoSignUp.tsx

```diff
@@ -0,0 +1,126 @@
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import habitsSuccessfulCustomer from 'sentry-images/spot/habitsSuccessfulCustomer.jpg';
+
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import HighlightCornerContainer from 'sentry/components/highlightCornerModal';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import {
+  extraQueryParameter,
+  extraQueryParameterWithEmail,
+  urlAttachQueryParams,
+} from 'sentry/utils/demoMode';
+
+type Props = ModalRenderProps;
+
+const NewDemoSignUpModal = ({closeModal}: Props) => {
```

Comment:
@roggenkemper I think it would be better to call this `DemoSignUpModalV2` (and use v2 instead of new everywhere else that's relevant). That way, the files are adjacent.

Reply:
sounds good! ill update this + the references in the demo repo too